docs/vc-rest/w3c_api_impl_status.md

```diff
@@ -0,0 +1,43 @@
+# W3C CCG HTTP API Implementation Status
```

Comment:
We might need a more generic title (and filename) as this document mixes both W3C CCG __and__ Transmute Extensions.

Reply:
any suggestions ?

Reply:
maybe `vc_interop_api` or something.

Reply:
[Done](https://github.com/trustbloc/edge-service/compare/9d085de3c4f9b8bdb2114e2c7ddcdda507501ce6..fcd5f59e27bafee8e169ab9bdf2ee79bc8731c19#diff-e3e8ccc1e3586fbcfa5cd223b35fcbdbR1),